ci/docker/tizen/prepare-sync.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+
+DEPS=src/flutter/DEPS
+
+if [ ! -f $DEPS ]; then
+    echo "Could not find DEPS file."
+    exit 1
+fi
+
+cat > .gclient << EOF
+solutions = [
+  { "name"        : 'src/flutter',
+    "url"         : 'https://github.com/flutter-tizen/engine.git',
+    "deps_file"   : 'DEPS',
+    "managed"     : False,
+    "custom_deps" : {
+    },
+    "custom_vars": {},
```

Comment:
```suggestion
    "custom_deps" : {},
    "custom_vars": {},
```
or you may remove these if they're not necessary.

Reply:
sure, I removed them.